ros2bag/ros2bag/verb/play.py

```diff
@@ -52,6 +52,10 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             '-l', '--loop', action='store_true',
             help='enables loop playback when playing a bagfile: it starts back at the beginning '
                  'on reaching the end and plays indefinitely.')
+        parser.add_argument(
```

Comment:
should we call this `--ros-args` (and call out that it enables topic name remapping in the `help` string), since you can pass any `ros-args` here and it'll work?

Reply:
Yes, we were wondering that. My reasoning not to go that route was the following:
- `--ros-args` was introduced in particular to avoid name clashes with user defined flags on user ros2 nodes, in other words to separate flags defined by rclcpp or rclpy, etc. from flags defined on nodes deriving from those
- Rosbag isn't the classical node and I don't see a scenario where you'd want to provide a custom rosbag node, so separating "namespaces" of flags is not a big deal here
- It could be interesting to expose more flags from rosbag's node, but I wonder whether it is a good idea to expose all of them. Therefore, I'd say: keep things hidden, so we'd only expose `--ros-args --remap` for now. I also wondered whether at that point it isn't confusing if the notation is the same for a general node but not all options you can generally pass to `--ros-args` are available.
What do you think about this?

Reply:
I have added added a new topic remapping options to the play options. Please let me know if there are still some other problems